packages/stylelint-config/rules/extras.js

```diff
@@ -11,7 +11,9 @@ module.exports = {
     'selector-nested-pattern': [
       '(^&:)|(^&\\[)|(^&\\.(is\\-|has\\-))',
       {
-        message: 'Only pseudo class selector, attribute selector and combination with state class is allowed',
+        message:
+          'Only pseudo class selector, attribute selector and combination ' +
```

Comment:
What about the usage of template literals? They are not allowed or do you see a different problem?

Reply:
No need for using literals. This string was split by mistake. Thought that linter doesn't like the lenght of the line but it was just a code style issue that the value of the message was previously not at its own line.